src/main/java/Parser.java

```diff
@@ -0,0 +1,85 @@
+public class Parser {
+    public static void addOrCompleteTask(String input, Task[] tasks){
+        String[] inputData = input.split(" ");
```

Comment:
Perhaps you can rename "inputData" array to something like "arguments", so as to adhere to the coding standard of 
"Plural form should be used on names representing a collection of objects."